File: decompiled/sources/com/streamax/config/listener/SuperListener.java
package com.streamax.config.listener;

public interface SuperListener {

    public interface GetConfigListener {
        void onFailure();

        void onSuccess(String str);
    }

    public interface LoadDeviceInfoListener {
        void onFailure();

        void onSuccess(Object obj);
    }

    public interface SetConfigListener {
        void onFailure();

        void onSuccess();
    }

    public interface SetRestoreDefaultListener {
        void onSuccess();
    }
}

File: decompiled/sources/com/streamax/config/listener/SuperListener.kt
package com.streamax.config.listener

interface SuperListener {
    interface GetConfigListener {
        fun onFailure()

        fun onSuccess(config: String)
    }

    interface LoadDeviceInfoListener {
        fun onFailure()

        fun onSuccess(deviceInfo: Any?)
    }

    interface SetConfigListener {
        fun onFailure()

        fun onSuccess()
    }

    interface SetRestoreDefaultListener {
        fun onSuccess()
    }
}
